content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -205,6 +214,121 @@ An [SQS Queue][32] (`aws_sqs_queue`) is considered publicly accessible if:
 
+## Azure Public accessibility logic by resource
+
+### Azure Network Security Group (NSG)
+
+An Azure NSG (`azure_security_group`) grants public access if:
+
+| Criteria | Explanation |
+|----------|-------------|
+|The security group has rules with protocol `tcp`, `udp` or `*` | These are the protocol values that are relevant for determining public access for the Azure resources that follow |
+|The security group has `inbound` rules with access set to `Allow` | Having these values indicates that the rule is allowing inbound traffic |
+|The security group has rules with source_address_prefix equal to `*`, `0.0.0.0`, `/0`, `::/0`, `internet`, or `any` | These CIDR prefixes allow access to the internet |
+|The rules which match the above properties combine with any other `Deny` rules of higher priority to open at least one port to the internet | See [here][39] for how Azure combines security group rules to calculate access |
+
+For more details on how Azure NSGs allow and deny internet access for a resource, see [here][40].
+
+### Azure Virtual Machine Instance
+
+A Virtual Machine Instance (`azure_virtual_machine_instance`) is considered publicly accessible if:
+
+* _Network Security Group allowing public access:_
+
+| Criteria | Explanation |
+|----------|-------------|
+|The virtual machine instance has a public IP address attached to its network interface | A public IP is required for internet access to a virtual machine instance |
+|The virtual machine instance has a network security group granting public access attached to its network interface | Refer to the Azure Network Security Group (NSG) section for how a network can grant public access |
+
+***OR***
+
+* _Public IP with SKU "Basic":_
+
+| Criteria | Explanation |
+|----------|-------------|
+|The virtual machine instance has a public IP address with SKU Basic attached to its network interface | A public IP address with SKU basic is open by default (see [here][41]) |
+|The virtual machine instance has no attached network security groups | If no network security groups are attached, then there are no rules blocking access through the open public IP address |
+
+Learn more about Azure Virtual Machine Instances and public access [here][42].
+
+### Azure Managed Disk
+
+A Managed Disk (`azure_managed_disk`) is considered publicly accessible if:
+
+| Criteria | Explanation |
+|----------|-------------|
+|The managed disk’s `network_access_policy` is set to `AllowAll` | This setting means that “The disk can be exported or uploaded to from any network” (see [here][43]) |
+|The managed disk’s virtual machine instance is publicly accessible | Refer to the Azure Virtual Machine Instance section above for how a virtual machine is considered publicly accessible |
+
+For more information about restricting import/export access for Azure Managed Disks, refer [here][44].
+
+### Azure Storage Account
+
+A Storage Account (`azure_storage_account`) grants public access if:
```

Comment:
for this one, IMHO this is an "internal attribute" that's used for public access determination of buckets (just like `aws_s3_account_public_access_block`) but shouldn't be documented, because it doesn't make a storage account public (it just doesn't _proactively block public access if a bucket in the storage account is set as public_)